File: src/main/java/frc/robot/commands/TareCmd.java
package frc.robot.commands;

import edu.wpi.first.wpilibj2.command.Command;
import frc.robot.subsystems.DriveSubsystem;

public class TareCmd extends Command {

    private final DriveSubsystem driveSub;

    public TareCmd(DriveSubsystem driveSub) {
        this.driveSub = driveSub;
        addRequirements(driveSub);
    }
    @Override
    public void execute() {
        this.driveSub.tare();
    }

    @Override
    public boolean isFinished() {
        return true;
    }
}

File: src/main/java/frc/robot/commands/TareCmd.kt
package frc.robot.commands

import edu.wpi.first.wpilibj2.command.Command
import frc.robot.subsystems.DriveSubsystem

class TareCmd(private val driveSub: DriveSubsystem) : Command() {
    init {
        addRequirements(driveSub)
    }

    override fun execute() {
        driveSub.tare()
    }

    override fun isFinished(): Boolean {
        return true
    }
}
